settings/eslint/eslintrc.js

```diff
@@ -0,0 +1,66 @@
+const commonParserOptions = {
+  ecmaFeatures: {
+    jsx: true,
+  },
+  ecmaVersion: '2020',
+  sourceType: 'module',
+};
+
+const commonRules = {
+  'prettier/prettier': [
+    'error',
+    {
+      singleQuote: true,
+      jsxSingleQuote: true,
+      trailingComma: 'all',
+    },
+  ],
```

Comment:
This is the only customization I did except applying the recommended presets below.